package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-sfd-landing-page",
-  "version": "0.4.3",
+  "version": "0.4.4",
```

Comment:
No actual changes are needed/being requested but the build kept failing on this branch. Will look into this but may take a couple of days while I get the frontend Helm updates working and merged in.

Reply:
Located the issue, `teamName` was missing from the `build.yaml` which was causing the failing build. Have added this in and once build is all complete will merge.